spec/discoverGranules/DiscoverGranules.input.template.json

```diff
@@ -0,0 +1,55 @@
+{
+  "cumulus_meta": {
+    "message_source": "sfn",
+    "state_machine": "",
+    "execution_name": ""
+  },
+  "meta": {
+    "workflow_name": "DiscoverGranules",
+    "stack": "",
+    "buckets": {
+      "internal": "",
+      "private": ""
+    },
+    "provider": {
+      "id": "http_provider",
+      "host": "http://ec2-54-159-27-56.compute-1.amazonaws.com:3030",
```

Comment:
How did you set this up? I think we should document this. Also I am curious.

Reply:
This is a temporary data endpoint set up (by @scisco), while we're waiting for https://bugs.earthdata.nasa.gov/browse/CUMULUS-465 to be unblocked.

Reply:
Ok cool. Can you please update 465 to make sure that this endpoint gets updated when that ticket is completed?